requirements.txt

```diff
@@ -0,0 +1,4 @@
+appdirs==1.4.0
```

Comment:
You're going to want to get used to virtual environments in python.  That's what this file does.  Right now your code is so simple it could run anywhere, but eventually you'll be including specific libraries that are versioned etc.